app/components/valid-input.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+    inputClassName: computed('className', 'errorMessage', {
+        get() {
+            let errorClass = this.errorClassName ? this.errorClassName : 'input-error'
```

Comment:
```suggestion
            const errorClass = this.errorClassName ? this.errorClassName : 'input-error'
```
unless the value of `errorClass` is going to change within the block use const.